catalog/transformer/catalog-transformer-tika-input/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD description="Tika Input Transformer"
+         name="Tika Input Transformer"
+         id="ddf.catalog.transformer.input.tika.TikaInputTransformer">
+
+        <AD description="Use the resource's metadata to determine the metacard title. If this is not enabled, the metacard title will be the file name."
+            name="Use Resource Title" id="useResourceTitleAsTitle" required="true" type="Boolean"
```

Comment:
I think it might be confusing to refer to the *resource* metadata's title and the *file* name in the same documentation. File and Resource are the same thing here. We should be consistent.

The code terminology is also ambiguous. 'ResourceTitle' really means 'ResourceMetadataTitle' as opposed to 'ResourceFilesystemTitle'. Recommend making this clearer and more consistent in the code so it's less confusing to maintain going forwards.